app/javascript/components/index.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
```

Comment:
- [ ] Change the name of this component to `app.js` since it is a requirement to have an app.js file that has the routes